src/config.zig

```diff
@@ -0,0 +1 @@
+pub const options = @import("../zig-cache/config_build_options.zig");
```

Comment:
Could this be made into a export const as so you wont have to pass an additional argument into every function that needs it, just extern it.

Reply:
So expand all the build options and export them with doc comments explaining what they do

Reply:
I think expanding them is a good idea. What's the benefit of using `extern` and `export` rather than just doing `config.rt_test`?

Reply:
Yeh, pass the config struct would be good